exporter/newrelicexporter/transformer.go

```diff
@@ -75,8 +79,9 @@ func newTraceTransformer(resource pdata.Resource, lib pdata.InstrumentationLibra
 	return t
 }
 
-func newLogTransformer(resource pdata.Resource, instrumentationLibrary pdata.InstrumentationLibrary) *logTransformer {
+func newLogTransformer(startInfo *component.ApplicationStartInfo, resource pdata.Resource, instrumentationLibrary pdata.InstrumentationLibrary) *logTransformer {
 	l := &logTransformer{
+		StartInfo: startInfo,
```

Comment:
Should we just append the collector name and version attributes to the resource attributes we create in this method? That would mean that we wouldn't have to create those attributes in each of the other methods, and they would get automatically applied.

Reply:
🏆 💡 Ooh good idea!!! 💥

Reply:
I just realized we can't do this because we want these attributes to override any span-specific attributes. If we were to put the attributes into the resources map, spans could override those attributes.